commons/src/main/java/org/springboardfoundation/usersservice/gender/Gender.java

```diff
@@ -0,0 +1,5 @@
+package org.springboardfoundation.usersservice.gender;
```

Comment:
No need to create a new package for gender.
The enum can be placed under org.springboardfoundation.usersservice.entity